doc/source/complex_composites.rst

```diff
@@ -0,0 +1,120 @@
+Complex Composites
+==================
+
+Many composites in |project| take advantage of various corrections or
+adjustments to produce the best looking imagery possible. The below
+sections describe the corrections and other related topics used in
+|project|. See the various :doc:`readers/index` documentation for more
+information on what products are available and descriptions of what
+corrections are used.
+
+.. _explain_rgb_composite:
+
+RGB Images
+----------
+
+An RGB image is an image constructed of three different color channels,
+Red, Green, and Blue. By interpreting the values from each channel and
+combining them we can produce an array of colors. This differs from a
+basic grayscale or "luminance" image where the single "L" channel
+determines how white or black a pixel is. For more information on the
+RGB color model see
+`this Wikipedia article <https://en.wikipedia.org/wiki/RGB_color_model>`_.
+
+You may also see RGBA images where the A stands for "Alpha". This Alpha
+channel will determine the opaque or transparency of an image. This is
+typically used in |project| to make invalid or missing data values
+transparent (completely opaque or completely transparent).
```

Comment:
@djhoese rather than describing the RGB color model how about something like:
Satellite imagers can simultaneously observe the Earth in multiple spectral channels, while the human eye is sensitive to only the visible channels. By mapping data from imager channels to the visible red, green and blue channels in different ways, we can produce “RGB” images that show the Earth as a human would see it from space (“true color”), or that emphasize certain features that can be detected using combinations of different channels (“false color”).

Reply:
That sounds good. I'll try to add something like that. I actually meant to mention satellite imagery here, but forgot after bouncing between sections. I'd still like to talk about RGB images (the color model) a small bit, but I'll see if I can make it fit better.